websubhub-ballerina/build.gradle

```
@@ -276,12 +286,21 @@ publishing {
     }
 }
 
-ballerinaBuild.dependsOn ":websubhub-native:build"
-ballerinaTest.dependsOn ":websubhub-native:build"
+ballerinaBuild.dependsOn ":${packageName}-native:build"
+ballerinaBuild.dependsOn updateTomlFile
+ballerinaBuild.dependsOn initializeVariables
+ballerinaBuild.dependsOn test
+
+ballerinaTest.dependsOn ":${packageName}-native:build"
+ballerinaTest.dependsOn updateTomlFile
+ballerinaTest.dependsOn initializeVariables
+
 unpackJballerinaTools.dependsOn copyToLib
 unpackStdLibs.dependsOn unpackJballerinaTools
 copyStdlibs.dependsOn unpackStdLibs
 updateTomlFile.dependsOn copyStdlibs
-ballerinaBuild.dependsOn updateTomlFile
-ballerinaBuild.dependsOn test
+
 build.dependsOn ballerinaBuild
```

test.dependsOn ballerinaTest
```suggestion
build.dependsOn ballerinaBuild
test.dependsOn ballerinaTest
```